bindings/nodejs/build.sh

```diff
@@ -122,12 +122,17 @@ cd $BUILD_ROOT
 test ! -d node_modules/nan && npm install nan@latest --save
 test ! -d node_modules/bindings && npm install bindings --save
 
+#   Still not sure of this
```

Comment:
That's a-ok, since we can always build the Electron binaries once the node binaries are in a good state.